test/filesystem.test

```diff
@@ -63,6 +63,14 @@ test_expect_success dd,mkdosfs,mcopy "vfat" "
 	check_filelist
 "
 
+exec_test_set_prereq mkfs.erofs
+exec_test_set_prereq fsck.erofs
+test_expect_success mkfs_erofs,fsck_erofs "erofs" "
+	run_genimage_root erofs.config test.erofs &&
+	fsck.erofs -p images/test.erofs | tee erofs.log &&
+	test_must_fail grep -q 'Filesystem was changed' erofs.log
```

Comment:
I'd like something to see that the content actually reached the filesystem.

1. use check_size
2. dump.erofs -s -S --ls --path=/ test.erofs | grep -v 'Filesystem created' > dump
   test_cmp "${testdir}/erofs.dump" "dump"

That will not check everything, but that contains the file count and the dir listing of the toplevel dir.

Tip: I insert a "cp dump ${testdir}/erofs.dump" so save the expected output on the first try and remove the line again afterwards.
And don't forget to add the new file to EXTRA_DIST in Makefile.am